packages/angular-ui-components/src/generated/component.ts

```diff
@@ -0,0 +1,120 @@
+/* tslint:disable */
```

Comment:
remove this whole directory (generated) as it's been renamed to stencil-generated and shouldn't be tracked.